v23.1/create-schedule-for-backup.md

```diff
@@ -63,7 +63,7 @@ Targets:
 `WITH SCHEDULE OPTIONS schedule_option` | Control the schedule behavior with a comma-separated list of [these options](#schedule-options).
 
 {{site.data.alerts.callout_info}}
-For schedules that include both [full and incremental backups](take-full-and-incremental-backups.html), CockroachDB will create two schedules (one for each type).
+For schedules that include both [full and incremental backups](take-full-and-incremental-backups.html), CockroachDB will create two schedules (one for each type). The [incremental backup schedule](#incremental-backup-schedules) follows the full backup. 
```

Comment:
nit: imo follows is a little ambiguous, can we clarify what you intended to convey by adding this line?

Reply:
Works for me. I'll link out to the section on the page.